src/cascadia/Remoting/WindowManager.cpp

```diff
@@ -509,4 +508,10 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
         _monarch.SummonWindow(args);
     }
 
+    Windows::Foundation::Collections::IMap<uint64_t, winrt::hstring> WindowManager::GetPeasantNames()
+    {
+        assert(_monarch);

```

Comment:
I'm guessing we won't want this. That said, maybe we should have an `if (_monarch)` check here?